geoportailv3/utils/db_to_pot.py

```diff
@@ -0,0 +1,22 @@
+from pyramid.paster import bootstrap
+env = bootstrap('development.ini')
+
+from c2cgeoportal.models import DBSession, TreeItem
+from os import path
+import codecs
+
+
+pathToPotFiles = './geoportailv3/locale/'
+p = path.join(pathToPotFiles, "geoportailv3-db.pot")
+
+if __name__ == "__main__":
+    w = codecs.open(p, 'w', encoding='utf-8')
+    w.write('#, fuzzy\nmsgid ""\nmsgstr ""\n"Plural-Forms: nplurals=2; \
```

Comment:
AFAIK, it is part of the specification of a pot / po file, and gettext tools will fail if not present.